### ISSUE TITLE
Lost all data. Import doesn't work

### ISSUE THREAD

**Issue by Aralox** [davidhealey/waistline#745]
v 3.7.2 installed via F-droid.
Been using this app for a while, perhaps a year. Opened it up yesterday, saw a black screen, force quit it and opened it again. All data gone. Tried to import from backup, but doesn't seem to do anything.
Uninstalled and installed via Google play (3.7.0), tried import again, no difference.
Happy to share the backup file with you privately.

**Comment by EmilJunker** [comment]
You can send the backup file to a8bfbe9842@outlook.com
I'll take a look at it.

**Comment by EmilJunker** [comment]
@Aralox Your backup file looks alright. I could import it without problems. What exactly happens when you try to import it? Clicking the Import From Backup button should open a file picker that lets you select the file for import. Does that not work for you?

**Comment by Aralox** [comment]
Thank you for looking at this Emil. It's actually working now. What was happening was when I clicked import and selected a file, it just returned to the import page without doing anything. I tried both 3.7.0 and 3.7.2. I reinstalled 3.7.2 and did the import again, and it worked. The only difference is that I used the default Android file picker instead of the 'solid Explorer' app, which I usually use for file stuff. Anyway happy to close this, unless you want to try and repro using solid Explorer. Thanks!
